src/constants.js

```diff
@@ -0,0 +1,15 @@
+
+
+export const routes = {
+	mainpage: '/',
+	card: '/card/:id',
+	linkCardId: (post) => {
+		return `/card/${post.id}`
+	}
+
+}
+
+export const classNameBootstrap = {
+	spinner: "spinner-border text-danger",
+	fixed: "sr-only"
+}
```

Comment:
Add eslint for the whole project and rule that automatically adds new empty lines to the end of file